### PR TITLE
add support for Fedora 34

### PR DESCRIPTION
**Pull request by ondrejbudai**
So far, this is a low effort PR to add support for Fedora 34. I've just copy-pasted all the required bits and generated test manifests. Let's see what happens on the CI.

The next step is obviously to adapt the manifests for changes in Fedora 34.

I think I've already found the first issue: I couldn't create the test manifest for IoT. Will investigate later.

This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change

### CODE REVIEW

File: test/data/repositories/fedora-34.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "x86_64": [
+    {
+      "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/public/f34/f34-x86_64-devel-20210414/",
```
Comment (henrywang) [inline test/data/repositories/fedora-34.json:4]: @ondrejbudai could you please add `name: xxxx` here and we can have source name in osbuild-composer source list. Thanks.

Comment (henrywang) [reply test/data/repositories/fedora-34.json:4]: And we need copy this file into `/etc/osbuild-composer/repositories/` in `provision.sh`

Comment (ondrejbudai) [reply test/data/repositories/fedora-34.json]: I force-pushed a fix, repositories now have names. The file should be already being copied, see: https://github.com/osbuild/osbuild-composer/blob/27c5aafeca102fb0634866c4adb2bb938d3262e7/tools/provision.sh#L52 I also verified this on Fedora 34 Schutzbot test machine:

```
$ ls /etc/osbuild-composer/repositories/
centos-8.json  centos-stream-8.json  fedora-32.json  fedora-33.json  fedora-34.json
```

**Comment by henrywang** [comment]
@ondrejbudai found the root cause. Could you please update line https://github.com/osbuild/osbuild-composer/blob/27c5aafeca102fb0634866c4adb2bb938d3262e7/test/cases/ostree.sh#L349
to `--extra-args="inst.ks=file:/ks.cfg console=ttyS0,115200"`

The `anaconda-34.24.9-1` in fedora 34 requires `inst` prefix.

**Pull request closed without merging**